Bio/AlignIO/NexusIO.py

```diff
@@ -113,17 +115,19 @@ def write_alignment(self, alignment, interleave=None):
         columns = alignment.get_alignment_length()
         if columns == 0:
             raise ValueError("Non-empty sequences are required")
-        minimal_record = "#NEXUS\nbegin data; dimensions ntax=0 nchar=0; " \
-                         + "format datatype=%s; end;"  \
-                         % self._classify_alphabet_for_nexus(alignment._alphabet)
+        minimal_record = (
+            "#NEXUS\nbegin data; dimensions ntax=0 nchar=0; "
+            + "format datatype=%s; end;"
+            % self._classify_alphabet_for_nexus(alignment._alphabet)
+        )
         n = Nexus.Nexus(minimal_record)
```

Comment:
Same here, the Strings above could be joined in one line.

Reply:
Also we don't need the plus sign.